src/test/resources/uk/gov/ons/ctp/journeys/send/survey/surveyReminders.feature

```diff
@@ -78,71 +82,81 @@ Feature: Tests the survey reminders are sent
     Then check "action.case" records in DB equal 497 for "actionplanfk = 1"
     And check "action.case" records in DB equal 3 for "actionplanfk = 2"
 
-
+# Skipping scenarios below as creating a BI case will not enrol respondent. Need to replace these steps using frontStage test scenarios. 
+# For now, updating this feature file to validate sending 'Survey Reminder Emails' for the existing 3 respondents only. 
+    
   # Create respondent case to send survey reminders to
 
-  Scenario: Verify event created for respondent enrolment
-    Given I make the POST call to the caseservice cases events
-      | Created by cucumber test | RESPONDENT_ENROLED | test | Cucumber Test |  |
-    When the response status should be 201
-    And the response should contain the field "createdDateTime"
-    And the response should contain the field "caseId"
-    And the response should contain the field "partyId"
-    And the response should contain the field "category" with value "RESPONDENT_ENROLED"
-    And the response should contain the field "subCategory" with value "test"
-    And the response should contain the field "createdBy" with value "Cucumber Test"
-    And the response should contain the field "description" with value "Created by cucumber test"
-    Then Check the case state has changed
-    And the response status should be 200
-    And the response should contain the field "state" with value "INACTIONABLE"
-
-  Scenario: Verify a new case have been created with the correct properties
-    Given after a delay of 60 seconds
-    When I make the GET call to the caseservice cases endpoint for new case
-    And the response status should be 200
-    Then the response should contain the field "id"
-    And the response should contain the field "state" with value "ACTIONABLE"
-    And the response should contain the field "iac"
-    And the response should contain the field "actionPlanId" with value "0009e978-0932-463b-a2a1-b45cb3ffcb2a"
-    And the response should contain the field "collectionInstrumentId"
-    And the response should contain the field "partyId"
-    And the response should contain the field "sampleUnitType" with value "BI"
-    And the response should contain the field "createdBy" with value "Cucumber Test"
-    And the response should contain the field "createdDateTime"
-    And the response should contain the field "responses" with one element of the JSON array must be []
-    And the response should contain the field "caseGroup"
-    And the response should contain the field "caseEvents" with one element of the JSON array must be [{"createdDateTime":
-    And the response should contain the field "caseEvents" with one element of the JSON array must be ,"category":"CASE_CREATED","subCategory":null,"createdBy":"SYSTEM","description":"Case created when Respondent Enroled"}
+#  Scenario: Verify event created for respondent enrolment
```

Comment:
I can't remember what was discussed sorry but were we going to use tags to skip scenarios, or were we just commenting out for now?

Reply:
@liamtoozer In native cucumber framework we do not get special tags to skip scenarios. However we could use ~<tag_name> within the Jenkins jobs to skip tags. For now, we are commenting out steps as it makes it consistent to feature files.